.github/workflows/nightlies.yml

```
@@ -0,0 +1,63 @@
+name: Upload nightly wheels to Anaconda Cloud
+
+on:
+  # Run daily at 1:23 UTC to upload nightly wheels to Anaconda Cloud
+  schedule:
+    - cron: '23 1 * * *'
+  # Run on demand with workflow dispatch
+  workflow_dispatch:
+
+jobs:
+  upload_nightly_wheels:
+    name: Upload nightly wheels to Anaconda Cloud
+    runs-on: ubuntu-latest
+    if: github.repository_owner == 'matplotlib'
+
+    steps:
+      - name: Install Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: '3.x'
+
+      # c.f. https://github.com/actions/download-artifact/issues/3#issuecomment-1017141067
+      - name: Download wheel artifacts from last build on 'main'
+        shell: bash
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          PROJECT_REPO="matplotlib/matplotlib"
+          BRANCH="main"
+          WORKFLOW_NAME="cibuildwheel.yml"
+          ARTIFACT_NAME="wheels"
+
+          gh run --repo "${PROJECT_REPO}" \
+             list --branch "${BRANCH}" \
+                  --workflow "${WORKFLOW_NAME}" \
+                  --json event,status,databaseId > runs.json
+          # Filter on 'push' events to main (merged PRs) that have completed
+          echo "$(cat runs.json | \
+            jq -c '[ .[] | select(.event == "push") | select(.status == "completed") ]'
+          )" > runs.json
+          # Get id of latest build of wheels
+          RUN_ID=$(cat runs.json | \
+            jq -c 'sort_by(.databaseId) | reverse | .[0].databaseId'
+          )
```

No need for the subshell, and now that I just checked, no need for `cat` either (alas I don't see a way to combine the two `select` into one or we might have made it under 80 even, but it's close enough.)
```suggestion
          jq --compact-output \
            '[ .[] | select(.event == "push") | select(.status == "completed") ]' \
            runs.json > pushes.json
          # Get id of latest build of wheels
          RUN_ID=$(
            jq --compact-output \
              'sort_by(.databaseId) | reverse | .[0].databaseId' pushes.json
          )
```
Also using the full argument name will be clearer for people.